tests/unit/api/main.py

```diff
@@ -572,7 +578,7 @@ def test_node_connect_network_no_such_node(self):
         network_create_simple('hammernet', 'anvil-nextgen')
 
         with pytest.raises(api.NotFoundError):
-            api.node_connect_network('node-98', '99-eth0', 'hammernet') # changed
+            api.node_connect_network('node-98', '99-eth0', 'hammernet')  # changed # noqa
```

Comment:
Quoting Nicholas Matsuura (2016-08-02 16:32:02)

>    You mean just move # changed to the line before? If so I can definitely
>    do that

I'm okay with this, but if you do this I'd like to add a colon, i.e.

    # changed:
    foo(...,...)

So that it's still clear what it's referring to.